dashboard-app/src/dashboard/ByActivity/index.tsx

```diff
@@ -29,13 +28,16 @@ const Container = styled(Grid)`
   width: 100% !important;
 `;
 
+const TABLE_TITLE = 'Volunteer time on activities';
+
 const ByActivity: FunctionComponent<RouteComponentProps> = (props) => {
   const [unit, setUnit] = useState(DurationUnitEnum.HOURS);
+  const [sortBy, setSortBy] = useState(`Total ${unit}`);
   const [activities, setActivities] = useState([]);
   const [volunteers, setVolunteers] = useState();
   const [fromDate, setFromDate] = useState(Months.defaultFrom());
   const [toDate, setToDate] = useState(Months.defaultTo());
-  const [tableProps, setTableProps] = useState<DataTableProps | null>();
+  const [tableProps, setTableProps] = useState<Pick<DataTableProps, 'headers' | 'rows'> | null>();
```

Comment:
This variable no longer represents table props. Can this be renamed to `tableRows` || `tableData` || somethings similar?